pkg/apis/datadoghq/v1alpha1/extendeddaemonset_types.go

```diff
@@ -80,18 +80,20 @@ type ExtendedDaemonSetSpecStrategyCanary struct {
 	NodeAntiAffinityKeys []string `json:"nodeAntiAffinityKeys,omitempty"`
 	// +optional
 	Paused bool `json:"paused,omitempty"`
+	// +optional
+	Failed bool `json:"failed,omitempty"`
```

Comment:
Is this `spec.canary` field used ?
I have the feeling that the status of the EDS is stored in `status.state` and the channel used by a user to force a failed state is annotations rather than this `spec` field.

Reply:
it's not! this is a relic from the first version, thanks for pointing it out